exampleCourse/questions/demoAutograderNumpy/tests/setup_code.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+import numpy.linalg as la
+
+# set up parameters
+n = np.random.randint(4, 16)
+A = np.eye(n)
+b = np.random.random(n)
+
+# hook into inverse functions
+def not_allowed(*args, **kwargs):
+    raise RuntimeError("Calling this function is not allowed")
```

Comment:
Should we provide `not_allowed()` as a library function, to avoid everyone needing to define it themselves?